ntbs-service/Services/TreatmentOutcomesHelper.cs

```diff
@@ -87,6 +87,10 @@ public static bool IsTreatmentOutcomeExpectedAtXYears(Notification notification,
             Notification notification,
             int numberOfYears)
         {
+            if (notification.IsPostMortemAndHasCorrectEvents && numberOfYears == 1)
+            {
+                return notification.TreatmentEvents?.Where(te => te.TreatmentEventTypeIsOutcome);
+            }
```

Comment:
I think it would be worth adding a comment here to explain that we're ignoring any diagnosis events that happen after death, because they shouldn't interfere with the outcome.